cpp/src/link_analysis/gunrock_hits.cu

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
Why can't this be header only?   Also, if not header,  since there are no cuda kernels here, why not .cpp

Reply:
Can't be header only.  We need to instantiate the template in order for it to be called from the python library.  I should be able to make it cpp.

Reply:
Changed it to a cpp